modules/gdscript/gdscript_parser.cpp

```diff
@@ -238,6 +238,7 @@ GDScriptParser::Node *GDScriptParser::_parse_expression(Node *p_parent, bool p_s
 	Node *expr = NULL;
 
 	int op_line = tokenizer->get_token_line(); // when operators are created at the bottom, the line might have been changed (\n found)
+	int next_valid_offset;
```

Comment:
Does this really need to be outside the loop?

Reply:
I guess not 🤔 
Would you rather have this only when it is used?

Reply:
Yes, since it clears up the logic; if the compiler feels like moving it out, it will.

Reply:
Ok, I changed it.